modules/priceFloors.js

```diff
@@ -332,20 +332,37 @@ export function getFloorDataFromAdUnits(adUnits) {
   }, {});
 }
 
+function getNoFloorSignalBidersArray(floorData) {
+  const { data, enforcement } = floorData
+  // The data.noFloorSignalBidders higher priority then the enforcment
```

Comment:
what's the reasoning for the duplicate option? I can think of why you might want to override it when you get it through dynamic `data`, but the priority should be inverted for that. What use case is this for?

Reply:
From the issue:
![image](https://github.com/prebid/Prebid.js/assets/13972794/e7768972-c96c-4af9-8f31-044250dff4a4)

If the dynamic or data object contains it, we should use what is set there, otherwise fallback.

If a pub uses some floor provider, the floor provider's dynamic endpoint may want the most control over this, which is why data is chosen first.

Reply:
I somehow missed that - thanks!